service/src/main/java/uk/gov/hmcts/reform/fpl/model/Direction.java

```diff
@@ -22,4 +25,14 @@
     private String directionNeeded;
     private String custom;
     private LocalDateTime dateToBeCompletedBy;
+    private DirectionResponse response;
+    private List<Element<DirectionResponse>> responses;
+
+    public List<Element<DirectionResponse>> getResponses() {
+        if (isNull(responses)) {
```

Comment:
could one line to `return isNull(responses) ? new ArrayList<>() : responses`

Reply:
I hate inline logic like this. But...
```
 defaultIfNull(responses, new ArrayList<>());
```

is a good shout